### PR TITLE
remove upper bound in setup.py

### PR DESCRIPTION
**Pull request by xiangyan99**
If a new version of our dependency is released, we need to make sure our library work with it.

Given new versions may contain security patches, we cannot stay with old version and stop our customers from using new ones.

### CODE REVIEW

**Review by mccoyp — APPROVED** [review]
Looks good for KV

**Comment by CFSNM** [comment]
Hey @xiangyan99, could we handle the Python 3.11 support for azure-keyvault in this PR?

https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/keyvault/azure-keyvault/setup.py

File: sdk/cognitivelanguage/azure-ai-language-conversations/setup.py [hunk 1]
```diff
@@ -75,8 +74,8 @@
         'azure.ai.language.conversations': ['py.typed'],
     },
     install_requires=[
-        "azure-core<2.0.0,>=1.28.0",
-        "isodate<1.0.0,>=0.6.1",
+        "azure-core>=1.28.0",
+        "isodate>=0.6.1",
```
Comment (kristapratico) [inline sdk/cognitivelanguage/azure-ai-language-conversations/setup.py:78]: should we keep upper bound for packages we don't own?

Comment (xiangyan99) [reply sdk/cognitivelanguage/azure-ai-language-conversations/setup.py:78]: No. We don't want the upper bound.

We cannot stop customers from using the latest one which may contain security patch.

**Review by kristapratico — APPROVED** [review]

**Pull request merged**